xml/System.Security/SecurityTransparentAttribute.xml

```diff
@@ -40,6 +40,8 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+ Partially trusted code is not longer supported. This attribute has no effect in .NET Core.
```

Comment:
@rpetrusha I would like to add this note to all attributes related to partially trusted code and code access security. Could you please take a look whether what I got is appropriate? Once we agree on the note text, I am going to copy&paste it into all affected attributes.

Reply:
Nit, not->no

Reply:
Text looks good to me now with the fix.